app/http.php

```diff
@@ -0,0 +1,346 @@
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use OpenRuntimes\Proxy\Health\Health;
+use OpenRuntimes\Proxy\Health\Node;
+use Swoole\Coroutine\Http\Client;
+use Swoole\Coroutine\Http\Server;
+use Swoole\Http\Request as SwooleRequest;
+use Swoole\Http\Response as SwooleResponse;
+use Swoole\Runtime;
+use Swoole\Table;
+use Swoole\Timer;
+use Utopia\App;
+use Utopia\Logger\Log;
+use Utopia\Logger\Logger;
+use Utopia\Logger\Adapter\AppSignal;
+use Utopia\Logger\Adapter\LogOwl;
+use Utopia\Logger\Adapter\Raygun;
+use Utopia\Logger\Adapter\Sentry;
+use Utopia\Balancing\Algorithm;
+use Utopia\Balancing\Algorithm\Random;
+use Utopia\Balancing\Algorithm\RoundRobin;
+use Utopia\Balancing\Balancing;
+use Utopia\Balancing\Option;
+use Utopia\CLI\Console;
+use Utopia\Registry\Registry;
+use Utopia\Swoole\Request;
+use Utopia\Swoole\Response;
+
+Runtime::enableCoroutine(true, SWOOLE_HOOK_ALL);
+
+App::setMode((string) App::getEnv('OPEN_RUNTIMES_PROXY_ENV', App::MODE_TYPE_PRODUCTION));
+
+// Setup Registry
+$register = new Registry();
+
+$register->set('state', function () {
+    $count = \count(\explode(',', (string) App::getEnv('OPEN_RUNTIMES_PROXY_EXECUTORS', '')));
+    $state = new Table($count);
+    $state->column('hostname', Swoole\Table::TYPE_STRING, 128); // Same as key of row
+    $state->column('status', Swoole\Table::TYPE_STRING, 8); // 'online' or 'offline'
+    $state->column('state', Swoole\Table::TYPE_STRING, 16384); // State as JSON
+    $state->create();
+    return $state;
+});
+
+$register->set('logger', function () {
+    $providerName = App::getEnv('OPEN_RUNTIMES_PROXY_LOGGING_PROVIDER', '');
+    $providerConfig = App::getEnv('OPEN_RUNTIMES_PROXY_LOGGING_CONFIG', '');
+    $logger = null;
+
+    if (!empty($providerName) && !empty($providerConfig) && Logger::hasProvider($providerName)) {
+        $adapter = match ($providerName) {
+            'sentry' => new Sentry($providerConfig),
+            'raygun' => new Raygun($providerConfig),
+            'logown' => new LogOwl($providerConfig),
+            'appsignal' => new AppSignal($providerConfig),
+            default => throw new Exception('Provider "' . $providerName . '" not supported.')
+        };
+
+        $logger = new Logger($adapter);
+    }
+
+    return $logger;
+});
+
+$register->set('algo', function () {
```

Comment:
```suggestion
$register->set('algorithm', function () {
```

the short form is okay when using it as a variable name , but when using it for classnames, or keys we should use the full form